homework3/test_hw_3/test_task03.py

```diff
@@ -0,0 +1,18 @@
+from homework3.task03.task03 import make_filter
+
+
+def test_make_filter_function_return_filtered_list():
```

Comment:
add more cases. what if there is not such data you'd like to filter by?
what if there are several lists?